modules/update-evaluate-availability.adoc

```diff
@@ -18,4 +18,4 @@ The CVO continuously evaluates its cluster characteristics against the condition
 If the CVO finds that the cluster does not match the risks of an update, or that there are no risks associated with the update, it stores the target version in the `availableUpdates` field of its `ClusterVersion` resource.
 
 The user interface, either the web console or the OpenShift CLI (`oc`), presents this information in sectioned headings to the administrator.
-Each *supported but not recommended* update recommendation contains a link to further resources about the risk so that the administrator can make an informed decision about the update.
+Each update recommendation for *known issues* contains a link to further resources about the risk so that the administrator can make an informed decision about the update.
```

Comment:
"for" -> "with"? Or maybe a bigger reshuffle to something like:

> Each known issue associated with the update path has a link...

Reply:
I like the latter because I think *technically* we aren't recommending this path :) good call